src/main/resources/db/migration/V1_95_3__referral_details_insert_data.sql

```diff
@@ -0,0 +1,14 @@
+insert into referral_details (id, superseded_by_id, created_at, created_by, reason_for_change, referral_id, completion_deadline, further_information, maximum_enforceable_days)
+      select
+            gen_random_uuid(),
+            null,
+            ref.created_at,
+            ref.created_by_id,
+            'initial value',
+            ref.id,
+            ref.completion_deadline,
+            ref.further_information,
+            ref.maximum_enforceable_days
+         from
+             referral ref
```

Comment:
Information in the copy will go stale if people continue to edit drafts and make referrals -- do we want to scope this to the unchangeable section (`sent_at NOTNULL`) first?

Reply:
We are now doing the population of that table in a separate piece of work